versions.gradle

```diff
@@ -3,16 +3,16 @@ ext {
             compileSdk : 28,
             minSdk     : 9,
             buildTools : '28.0.3',
-            versionName: '0.3.5'
+            versionName: '0.3.6'
     ]
 
-    final supportLibraryVersion = '28.0.0'
+    final supportLibraryVersion = '1.0.2'
```

Comment:
I think this should be explicit with the `androidx` packages since the version of artifacts are unbundled unlike the previous version scheme of support libraries. This based on looking at [AndroidX release versions](https://developer.android.com/jetpack/androidx/versions/). Even though currently `appcompat` and `annotation` modules share the same version, it might diverge in next updates.